src/components/challenges/active.tsx

```diff
@@ -33,38 +31,54 @@ type ActiveProps = {
 const Active: FC<ActiveProps> = ({ content }) => {
   const [isUserSubscribed, setUserSubscribed] = useState(content.is_user_subscribed);
 
-  const subscribeEventHandler = async () => {
-    setUserSubscribed(1);
+  const taskData = {
+    Level: content.level,
+    Challenge_Started: content.start_date,
+    Challenge_Ends: content.end_date,
+    Active_Participants: content.participants.length,
   };
 
+  const task: any[] = [];
+  function getTask() {
+    // eslint-disable-next-line array-callback-return
+    Object.entries(taskData).map(([key, value]) => {
+      task.push({ key, value });
+    });
+    return task;
+  }
+
+  const arrayOfParticipants: any[] = content.participants;
+
+  const participants: any[] = [];
+  function getParticipants(users: any[]) {
+    // eslint-disable-next-line array-callback-return
```

Comment:
can we please get rid of this?

Reply:
Used forEach instead of map to get rid of this comment.

https://github.com/Real-Dev-Squad/website-status/commit/f84f9afa7df2679012784eb4f1b1558a2496fdb4